libs/system-admin-pages/src/pages/reference-data-upload/index.njk

```diff
@@ -0,0 +1,70 @@
+{% extends "layouts/base-template.njk" %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/file-upload/macro.njk" import govukFileUpload %}
+{% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
+
+{% block content %}
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+
+    {% if errors %}
+      {{ govukErrorSummary({
+        titleText: errorSummaryTitle,
+        errorList: errors
+      }) }}
+    {% endif %}
+
+    <div style="background-color: #f0f4f5; padding: 20px; margin-bottom: 30px;">
```

Comment:
Can we move the styling to the CSS file and include the styling in a 'class' attribute here